tests/src/test/scala/whisk/core/entity/test/ExecHelpers.scala

```diff
@@ -99,4 +99,6 @@ trait ExecHelpers extends Matchers with WskActorSystem with StreamLogging {
   protected def bb(image: String, code: String, main: Option[String] = None) = {
     BlackBoxExec(ExecManifest.ImageName(trim(image)), Some(trim(code)).filter(_.nonEmpty), main, false)
   }
+
+  protected def getRuntimesManifest() = ExecManifest.runtimesManifest
```

Comment:
Should drop the "get" and `()` since this is just an accessor?

Reply:
Why not just use ExecManifest.runtimesManifest
directly?

Reply:
I believe `ExecManifest.runtimesManifest` can only be accessed from packages in the `core`. It is defined with an access modifier `protected[core] object ExecManifest {`.

Reply:
ok - rather than add another get method to change the package protection, widen it at the source in that case.